app/services/data_migrations/fix_lat_long_flip_flops.rb

```diff
@@ -0,0 +1,36 @@
+module DataMigrations
+  class FixLatLongFlipFlops
+    TIMESTAMP = 20210323203521
+    MANUAL_RUN = false
+
+    def change
+      audits = Audited::Audit.where(
+        "auditable_type = 'Provider' AND
+        action = 'update' AND
+        audited_changes ?& array['latitude', 'longitude'] AND
+        (audited_changes - 'latitude') - 'longitude' = '{}'",
+      ).order(created_at: :asc)
+
+      audit_count = 0
+      total_audits = audits.count
+
+      Provider.find_each do |p|
+        audits_for_provider = audits.where(auditable_id: p.id)
+
+        # if we already had lat/lng when we made the first spurious update, delete every audit
+        if audits.first.audited_changes['latitude'].last.nil?
```

Comment:
```suggestion
        if audits_for_provider.first.audited_changes['latitude'].last.nil?
```
Not sure about this, but `audits.first` would always be the first of the audits for all providers I think?

Reply:
Good catch!